crates/rpc/rpc-engine-api/src/error.rs

```diff
@@ -20,20 +20,20 @@ pub const REQUEST_TOO_LARGE_CODE: i32 = -38004;
 #[derive(Error, Debug)]
 pub enum EngineApiError {
     /// Unknown payload requested.
-    #[error("Unknown payload")]
+    #[error("unknown payload")]
```

Comment:
must be uppercase

Reply:
Makes sense, is there something like this for RPC API errors as well?